test/qa/lib/meshnet.py

```diff
@@ -666,3 +667,28 @@ def is_connect_successful(output:str, peer_hostname: str):
 def get_lines_with_keywords(lines: list[str], keywords: list[str]) -> list:
     """Returns list with elements, that contain specified `keywords`."""
     return [line.strip() for line in lines if all(keyword in line for keyword in keywords)]
+
+def are_peers_connected(ssh_client: ssh.Ssh = None, retry: int = 3) -> None:
+    """
+    Verifies if local and remote NordVPN mesh peers see each other as connected in peer list.
+
+    Args:
+        ssh_client (ssh.Ssh): SSH client to execute commands on the remote system.
+
+    Raises:
+        pytest.fail: If peers are not connected after `retry` attempts.
+    """
+
+    for refresh_count in range(retry):
```

Comment:
Shouldn't there be a delay not to have issues with the API?

Reply:
Good point, I did not think about this. Would you agree, that a delay of a second or two should be enough?

Reply:
yeah, 1-2 seconds should be fine